src/helpers/isHostElementPointerEventEnabled.ts

```diff
@@ -0,0 +1,26 @@
+import { ReactTestInstance } from 'react-test-renderer';
+import { getHostParent } from './component-tree';
+
+// pointerEvents controls whether the View can be the target of touch events.
+// 'auto': The View can be the target of touch events.
+// 'none': The View is never the target of touch events.
+// 'box-none': The View is never the target of touch events but its subviews can be
+// 'box-only': The view can be the target of touch events but its subviews cannot be
+// see the official react native doc https://reactnative.dev/docs/view#pointerevents
+export const isHostElementPointerEventEnabled = (
```

Comment:
The current version of `isPointerEventEnabled` is essentially the same, so you should be able to use it.

Reply:
I kept a single version of it